faucet/faucet_dot1x.py

```diff
@@ -56,12 +56,30 @@ def auth_handler(self, address, _group_address):
         self.logger.info(
             'Successful auth from MAC %s on %s' % (
                 str(address), self.dot1x_port))
-        self._valve._inc_var('dot1x_success') # pylint: disable=protected-access
+        self.metrics.dp_dot1x_success.labels(**self._valve.base_prom_labels).inc()
```

Comment:
Not sure how these should be done (protected-access). (_inc_var looks tidier)

Reply:
maybe part of the functionality of _inc_var should be moved to a function in the metrics object to make this a little cleaner. But I think it is a little weird for dot1x to ask valve to update its metrics for it.

Reply:
moved part of _inc_var the metrics object in commit 8dc8c9c